### DISCUSSION TITLE
绳网无法进入（已解决）

### DISCUSSION THREAD

**Discussion by RS64671012** [share121/inter-knot#665] [崩坏：星穹铁道]
Q&A：<font color="green">answered</font>
![image](https://github.com/user-attachments/assets/20ef6166-d7c5-42b1-ba00-4805e3497889)
![image](https://github.com/user-attachments/assets/726832b7-e527-4351-a22d-e92160c8d319)
> 如图，进入时显示需登录，然已登录Github，且已更新，求解决！
> （edge浏览器，steam++加速）

已解决

**Comment by Yizakl** [comment]
点左上角授权

**Comment by RS64671012** [comment]
> 点左上角授权

未解决，网址栏后出现"?code=xxx"型字符

**Comment by NIUTE-SKMD** [comment]
同问cy

**Comment by BlakrPander** [comment]
同问cy

**Comment by RS64671012** [comment]
>未解决，网址栏后出现"?code=xxx"型字符

补充：ctrl+R刷新xxx不变，点击蓝字和头像xxx变化

**Reply by Yizakl** [reply]
这我也不清楚，我也不太懂

**Comment by share121** [comment]
现在，再试试

**Comment by Yizakl** [comment]
脚本更新了修了这个bug吗

**Comment by share121** [comment]
是的，我这边已经没有问题了

**Comment by RS64671012** [comment]
> 现在，再试试

仍然未解决

**Comment by BlakrPander** [comment]
没变 还是同楼主一样的问题

**Reply by BlakrPander** [reply]
目前的code：d7536d634ec35cbf7b06

**Reply by RS64671012** [reply]
> 目前的code：d7536d634ec35cbf7b06

code会变

**Comment by Dave-12138** [comment]
![image](https://github.com/user-attachments/assets/29731c25-58ff-4270-a932-12056c559937)
因为Manifest 404了？

**Reply by RS64671012** [reply]
是的

**Comment by Lanmuqiao** [comment]
加1，我也是这样，进不去

**Comment by sekiroissekiko** [comment]
我也进不去，插件也更新了但一直卡在请登录后查看那里

**Comment by RS64671012** [comment]
目前已解决
